src/ios/VLCPlugin.m

```diff
@@ -72,6 +79,8 @@ @interface VLCPlugin ()
 @property NSURLSessionDataTask *lockscreenImageDownloadTask;
 @property NSTimer *heartbeatTimer;
 @property NSTimeInterval lastHeartbeat;
+@property VLCPluginMediaType activeMediaType;
+@property NSString *activePK;
```

Comment:
We should avoid utilizing a NYPR-specific field such as PK here, as this is an open-source plugin.